src/DynamoCore/DynamoCore.csproj

```diff
@@ -29,11 +31,14 @@
   <ItemGroup Label="Common dependencies">
     <PackageReference Include="Autodesk.IDSDK" Version="1.1.6" />
     <PackageReference Include="DynamoVisualProgramming.Analytics" Version="3.0.0.2506" />
-    <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.17.1.3967" GeneratePathProperty="true"/>
-    <PackageReference Include="DynamoVisualProgramming.LibG_228_0_0" Version="2.17.1.3967" GeneratePathProperty="true"/>
+    <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.18.0.646" GeneratePathProperty="true"/>
+    <PackageReference Include="DynamoVisualProgramming.LibG_228_0_0" Version="2.18.0.654" GeneratePathProperty="true"/>
     <PackageReference Include="Greg" Version="2.3.0.2505" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" CopyXML="true" />
 	  <PackageReference Include="RestSharp" Version="106.12.0" CopyXML="true" />
+    <Reference Include="System.Collections.Immutable">
```

Comment:
what is this change for?

Reply:
it looks quite strange - it's a reference to `System.Collections.Immutable` - but pointing at Protogeometry?

Reply:
That was supposed to be ProtoGeometry

Reply:
Ugh..I see now that indirect references are not turned on..